plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryPager.php

```diff
@@ -49,123 +49,29 @@ protected function get_block_type_uses_context() {
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
-		$pager_display_mode = $block->context['pagerDisplayMode'] ?? '';
-
-		if ( 'off' === $pager_display_mode ) {
-			return null;
-		}
-
-		$number_of_thumbnails = $block->context['thumbnailsNumberOfThumbnails'] ?? 0;
-		$wrapper_attributes   = get_block_wrapper_attributes();
-		$post_id              = $block->context['postId'] ?? '';
-		$product              = wc_get_product( $post_id );
+		$post_id = $block->context['postId'] ?? '';
+		$product = wc_get_product( $post_id );
```

Comment:
if `$post_id` is `''` is there any point in executing `wc_get_product`? We could just immediately return `''` I guess?